packages/css/src/components/search-bar/search-bar.scss

```diff
@@ -1,5 +1,9 @@
+@import "../../../../sources/src/styling/variables";
+
 $dso-search-bar-height: $u5;
 $dso-search-bar-invalid-input-border-color: $dso-danger-color;
+$dso-search-bar-invalid-border-color: $dso-danger-color;
+$dso-search-bar-focus-border-width: 1px;
```

Comment:
Ik kan niet zo terugvinden waar deze variablen worden gebruikt, mis ik iets?

Reply:
Dat is legacy van mijn vorige constructie: ik haal het weg.